style.css

```diff
@@ -0,0 +1,169 @@
+@media only screen and (min-width: 768px) {
+  
+  
+}
+@media only screen and (min-width: 375px) {
+  
+  
+}
```

Comment:
- [ ] Kindly omit this media query here as it is already set to all screens if you do not define them. You are only going to need the 'min-width: 768px' query later for responsibility.